### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by upexcel**

### CODE REVIEW

File: src/app/config/config.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
+export const config = {
+    'interviewRounds': [{ 'name': 'First Round', 'value': 'first_round' }, { 'name': 'Second Round', 'value': 'second_round' }, { 'name': 'Third Round', 'value': 'third_round' }],
```
Comment (reactexcel) [inline src/app/config/config.ts]: what is this entire config for can you write comments in front of each

File: src/app/modules/pages/tag-settings/tag-settings.service.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  providedIn: 'root'
+})
+export class TagSettingsService {
+
+  constructor(
+    private http: HttpClient,
+  ) { }
+
+  getAllTag(){
+    return this.http.get(`${environment.apibase}/tag/get`).toPromise();
```
Comment (reactexcel) [inline src/app/modules/pages/tag-settings/tag-settings.service.ts]: does angular still work on promise? and not async/await? ask prateek if needed

File: src/app/modules/pages/tag-settings/tag-settings.component.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  ngOnInit() {
+    this.loading = true;
+    this.getAllTag();
+  }
+
+  async getAllTag(){
+      try{
+          const tag = await this.tagsettingservice.getAllTag();
+          this.formatTagsInArray(tag)
+}catch(error){
```
Comment (reactexcel) [inline src/app/modules/pages/tag-settings/tag-settings.component.ts]: code formatt

File: src/app/modules/pages/tag-settings/tag-settings.service.ts (hunk 2 above)
Comment (upexcel) [reply src/app/modules/pages/tag-settings/tag-settings.service.ts]: yes angular still uses both.

File: src/app/modules/components/sidenav/sidenav.component.html [hunk 4]
```diff
@@ -1,149 +1,136 @@
...
+            <span class="circle bg-warning fw-bold text-gray-dark">
         13
-      </span>
-      &nbsp; Philip
-      <strong>Smith</strong>
-      <b class="caret"></b>
-    </a>
-    <!-- <app-notifications *dropdownMenu class="dropdown-menu dropdown-menu-left animated animated-fast fadeInUp">
+      </span> &nbsp; Philip
+            <strong>Smith</strong>
```
Comment (reactexcel) [inline src/app/modules/components/sidenav/sidenav.component.html]: what is this philp smith?

File: src/app/modules/components/sidenav/sidenav.component.html [hunk 5]
```diff
@@ -1,149 +1,136 @@
...
-      <ul id="sidebar-interview-test-system" class="collapse">
+    </div>
+    <!-- main notification links are placed inside of .sidebar-nav -->
+    <ul class="sidebar-nav">
         <li>
-          <a [routerLink]=" ['interview-test-system/product-grid'] ">Candidate Score</a>
+            <a [routerLink]=" ['/dashboard/company-profile'] ">
+                <span class="icon">
+          <i class="fi flaticon-calculator-1"></i>
+        </span> Company Profile
```
Comment (reactexcel) [inline src/app/modules/components/sidenav/sidenav.component.html]: code formatting very poor

Comment (reactexcel) [reply src/app/modules/components/sidenav/sidenav.component.html]: format it

File: src/app/modules/pages/tag-settings/tag-settings.service.ts (hunk 2 above)
Comment (reactexcel) [reply src/app/modules/pages/tag-settings/tag-settings.service.ts]: ok

File: src/app/modules/components/sidenav/sidenav.component.html (hunk 4 above)
Comment (upexcel) [reply src/app/modules/components/sidenav/sidenav.component.html]: its from the design template, will remove it when we have the real name.

**Pull request merged**